accepted/2021/x64-emulation-on-arm64/x64-emulation.md

```diff
@@ -245,3 +245,34 @@ Actual Plan:
 We decided that option 3 will result in CLI breaking changes. While option 2 could theoretically be delivered between .NET 6 and .NET 7 with a .NET SDK update (like `6.0.200`), option 3 can only be delivered with a major release. It is also fair to note that the breaking changes associated with option 3 are not dependent on option 2. We could in theory make the breaking changes for option 3 now, enabling the remaining work to be done either before .NET 6 or in a .NET SDK update.
 
 Note: These plans assume that Visual Studio (Dev17) will not be supported on Windows Arm64 (with x64 emulation) before .NET 7. They also assume that Visual Studio for Mac will adapt to this plan, aligned with .NET 6. If those are not true, then we need to re-assess this plan.
+
+## Update
+
+Update: September 23, 2021
+
+The following is what we've decided to support, in terms of x64 emulation and native Arm64 architecture builds and their necessary co-existence.
+
+Supported products:
+
+- .NET Core 3.1 x64 runtimes
+- .NET 5 x64 runtimes
+- .NET 5 Arm64 runtimes (Windows only)
+- .NET 6+ x64 and Arm64 runtimes
+- .NET 6+ x64 and Arm64 SDKs
+
```

Comment:
It's important to note that the 3.1 x64, 5.0 x64, and 5.0 ARM64 SDKs will not block installation on ARM64 but they are not present in this list.  This means that they are all "unsupported" in their latest serviced version and folks should avoid installing them.

It's also important to note that versions of all these products prior to fixes will not be supported and should be uninstalled manually.

Reply:
Excellent points.

Reply:
https://github.com/dotnet/designs/pull/249/commits/28cf293b4a6d6c628ab5932382cb05eab4b2a91c